src/main/java/duke/commands/CreateEvent.java

```diff
@@ -0,0 +1,50 @@
+package duke.commands;
+
+import java.io.IOException;
+
+import duke.ui.Ui;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.tasks.Event;
+import duke.storage.Storage;
+import duke.parsers.DateTimeParser;
+import duke.exceptions.DukeException;
+
+/**
+ * Creates an Event Task and adds it to the TaskList.
+ */
+class CreateEvent extends TimedCommand {
+
+    public CreateEvent(DateTimeParser dtParser) {
+        super(dtParser);
+    }
+
+    public void execute(String arg, TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        // Perform parsing of arguments
+        String[] args = arg.split("/at");
```

Comment:
This is a lower-level operation, effectively reducing SLAP in this method. Consider moving this out to a separate method, thereby improving SLAP